internal/app/component_version/component_version_handler.go

```diff
@@ -117,6 +118,9 @@ func (cv *componentVersionHandler) CreateComponentVersion(componentVersion *enti
 	newComponent, err := cv.database.CreateComponentVersion(componentVersion)
 
 	if err != nil {
+		if strings.HasPrefix(err.Error(), "Error 1062") {
```

Comment:
See my other comment regarding errors from the db

Reply:
Agree